### ISSUE TITLE
[BUG] - Resource `mgc_network_public_ips` is bogus

### ISSUE THREAD

**Issue by nawarian** [MagaluCloud/terraform-provider-mgc#48]
**Describe the bug**

The registry's docs have the [mgc_network_public_ips resource](https://registry.terraform.io/providers/MagaluCloud/mgc/latest/docs/resources/network_public_ips) listed, but it doesn't really exist.

A quick check on tofu's fetched schema shows that the resource is not available.
```
tofu providers schema -json | jq .
```

This is rather important because without this resource I can't attach a public_ip to my vpc.

**To Reproduce**

```
// network.tf
resource "mgc_network_vpc" "codamos_vpc" {
  name = "codamos-vpc"
}

resource "mgc_network_public_ips" "codamos_public_ip" {
  vpc_id = mgc_network_vpc.codamos_vpc.id
}
```

```
$ tf apply
╷
│ Error: Invalid resource type
│ 
│   on network.tf line 5, in resource "mgc_network_public_ips" "codamos_public_ip":
│    5: resource "mgc_network_public_ips" "codamos_public_ip" {
│ 
│ The provider registry.terraform.io/magalucloud/mgc does not support resource type "mgc_network_public_ips".
╵
```

**Expected behavior**

Resources should be available as described by the docs.

**Desktop:**
 - OS: Ubuntu 22.04 LTS
 - Version 0.26.2

**Comment by nawarian** [comment]
Moving forward with my tests I've also found that most things related to networking are documented but not yet available, although their respective `mgc` CLI commands seem to work fine.

I see resourecs like `mgc_network_public_ips`, `mgc_network_rules` or `mgc_network_security_groups` available via documentation but the terraform schema doesn't really offer the resource.

This got me very confused and frustrated. But I appreciate this is still in its early stages, so I'll just do some workarounds via CLI and keep moving forward.

**Comment by publi0** [comment]
Solved on the latest release v0.29.2
